docs/secondary-network-ipam.md

```diff
@@ -0,0 +1,161 @@
+# Antrea IPAM for Secondary Network
+
+With the [AntreaIPM feature](antrea-ipam.md#antrea-flexible-ipam), Antrea can
+allocate IPs for Pod secondary networks. At the moment, AntreaIPAM supports
+secondary networks managed by [Multus](https://github.com/k8snetworkplumbingwg/multus-cni),
+we will add support for secondary networks managed by Antrea in future.
```

Comment:
I may be missing something, but that seems in direct contradiction with what you added to the config comments:
> It is also required by secondary network IPAM

I feel like it is quite confusing to use the "secondary network" terminology in both cases, as this is the name of the Antrea feature.

Reply:
What is your suggestion then? But at least when we finally support IPAM for Antrea native secondary network, then it will not be problem any more?

Reply:
I guess most of my confusion comes from the "It is also required by secondary network IPAM." statement in other places. That doesn't seem to apply to either case:
1) For the Multus case, any IPAM solution can be used, so AntreaIPAM is not _required_
2) For the Antrea SecondaryNetwork feature, only Whereabouts is supported at the moment

I think I see what you are trying to say, but maybe something like this would help clarify (in config comments):
> To use Antrea for IPAM when configuring secondary network interfaces with Multus, this feature gate needs to be enabled.

For this documentation, the current version sounds good to me. In the last sentence I would just add a reference to the Antrea feature name (`SecondaryNetwork`) for clarification.

BTW, I am not sure why we need a new document and cannot place this information in https://github.com/antrea-io/antrea/blob/main/docs/antrea-ipam.md?

Reply:
Ok. Let me rephrase the comments on feature gate. I can merge the doc to antrea-ipam.md as well. Earlier thought secondary network IPAM is not interested by most people, so created a separate doc.

Reply:
Updated.